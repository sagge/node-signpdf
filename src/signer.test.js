import SignPdfError from './SignPdfError';
import Signer from './signer';

describe('Test signing', () => {
    it('sign method must be implemented', async () => {
        try {
            const signer = new Signer(Buffer.from(''));
            await signer.sign('non-buffer');
            expect('here').not.toBe('here');
        } catch (e) {
            expect(e instanceof SignPdfError).toBe(true);
            expect(e.type).toBe(SignPdfError.TYPE_INPUT);
            expect(e.message).toMatchSnapshot();
        }
    });
});
