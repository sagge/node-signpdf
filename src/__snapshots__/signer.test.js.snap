// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test signing sign method must be implemented 1`] = `"sign not implemented on Signer"`;
